tests/Backend/Snowflake/OrderByTest.php

```diff
@@ -136,15 +136,26 @@ public function testNonArrayParamsShouldReturnErrorInAsyncExport()
     {
         $tableId = $this->prepareTable();
 
-        $params = [
```

Comment:
Padá ti test testNonArrayParamsShouldReturnErrorInDataPreview

Reply:
```
Failed asserting that exception message 'All items in param "orderBy" should be an arrays, but request contains:
Array
(
)
' contains 'Parameter orderBy must be array. Like &orderBy[0][column]=...'.
```

Reply:
Mimochodem - přijde mi zvláštní i ta samotná chyba, protože v testu je 

```
$params = [
            'orderBy' => ['column' => 'column'],
        ];
```

A v error message je prázdné pole...

Reply:
áá sorry, to jsem přehlídnul. Zapomněl jsem, že tam jsou testy jak na async export, tak datapreview. Doplnil jsem ještě oddělené testy (datapreview + async export) do testu na whereFilter.

Reply:
To pořád nevysvětluje proč je tam 
```
All items in param "orderBy" should be an arrays, but request contains:
Array
(
)
```

Tj. proč tam není v té error message `'orderBy' => ['column' => 'column'],`

Reply:
Jo už to vidím. Ono to vrací `$_POST`, ale ty parametry se posílájí v `$_GET`. Není mi jasný, proč jsem tam napsal `$_POST` :confused: . No zase se ukazuje, že je lepší kontrolovat celou exception message :)

Reply:
Nemělo by tam být IMHO ani POST ani GET. Mělo by tam být to, co validuješ...

Reply:
Jo tak už vím. Ono totiž exportovací job se volá skrze `POST`, ale datapreview skrze `GET`! ... hmm takže tam bude potřeba logika, který to vezme podle typu requestu...

```
$method = '_' . $_SERVER['REQUEST_METHOD'];
print_r($$method);
```
:heart_eyes:

Reply:
Tak super tríček s `$$` u globálních proměnných nefunguje. Musel jsem tam udělat trapně `switch` :-1: